VmAgent.Core/Interfaces/ProcessRunner.cs

```diff
@@ -76,6 +79,27 @@ public override Task CollectLogs(string processId, string logsFolder, ISessionHo
             // The game server is free to read the env variable for log folder and write all output to a file in that folder.
             // If required, we can add action handlers (see SystemOperations.RunProcess for example).
             // However, keeping the file handle around can be tricky.
+            throw new NotSupportedException();
+        }
+
+        public override Task CreateExceptionLogs(string logsFolder, string exceptionMessage)
```

Comment:
This method is to be used only when the startup of the game server fails, correct? Maybe give it a more suitable naming then.